corehq/apps/reports/tests/readable_forms/mismatched_group_hierarchy.result.yaml

```diff
@@ -7,7 +7,8 @@
   label: You first need to log a facility before you can record information.
   label_ref: need_facility_trigger-label
   options:
-  - {label: Go back and exit form, value: exit_form}
+  - {label: Go back and exit form, label_ref: need_facility_trigger-exit_form-label,
```

Comment:
You had to manually add all these?? :/

Reply:
It's not totally manual. when the test fails it gives you some message that shows you how to print out hte correct yaml file https://github.com/dimagi/commcare-hq/blob/3a75291943361657e39af72270b3068743f79928/corehq/apps/reports/tests/test_readable_formdata.py#L273-L281